### PR TITLE
Add timestamp to every error so we can trace them

### PR DESCRIPTION
**Pull request by danielgomezrico**
This may help to trace the errors back to logs and find fixes easy.

```
{
    "errors": [
        {
            "title": "Error",
            "message": "Something happened but our team is already working on it.",
            "timestamp": "2018-04-13T00:27:28.240Z"
        }
    ]
}
```

### CODE REVIEW

File: backend/API_standards.md [hunk 1]
```diff
@@ -174,11 +176,13 @@ Validation error example:
     {
       "title": "Invalid format",
       "message": "The email haves an invalid format",
+      "timestamp": "2018-03-12T18:30:00.SSSZ"
```
Comment (orendon) [inline backend/API_standards.md]: missing `,` at the end of the line

File: backend/API_standards.md [hunk 2]
```diff
@@ -174,11 +176,13 @@ Validation error example:
...
       "attribute": "email"
     },
     {
       "title": "Invalid format",
       "message": "The password length should be at least 8 characters",
+      "timestamp": "2018-03-12T18:30:01.SSSZ"
```
Comment (orendon) [inline backend/API_standards.md]: same here, `,`

File: backend/API_standards.md [hunk 3]
```diff
@@ -161,7 +162,8 @@ Not Found example:
   "errors": [
     {
       "title": "Resource not found",
-      "message": "User not found, please check your identifier"
+      "message": "User not found, please check your identifier",
+      "timestamp": "2018-03-12T18:30:00.SSSZ"
```
Comment (orendon) [inline backend/API_standards.md:166]: We currently have timestamp on backend and Papertrail logs, can you tell a bit more about the benefits of having this as an API consumer?

**Review by orendon — COMMENTED** [review]
Nice! I just want to dig more into the benefits/usage of this

File: backend/API_standards.md (hunk 3 above)
Comment (danielgomezrico) [reply backend/API_standards.md:166]: @orendon when a user reports an error normally he shows the error that he got from backend.
If we have the `timestamp` info in that report we can trace more easily when it appeared, and then match which logs on papertrail corresponds with the error.

What do you mean by "We currently have timestamp on backend"? where is that?

Comment (danielgomezrico) [reply backend/API_standards.md:166]: Having this already may help us to debug our apps, see @lordaniel asking me for the time when the error occurred as a real life example:

screen shot 2018-05-03 at 1 24 42 pm

**Comment by danielgomezrico** [comment]
This makes no sense since the date is already on the response as a header.

**Pull request closed without merging**